generators/entity/templates/client/angular/src/main/webapp/app/entities/_entity-management-detail.component.ts

```diff
@@ -8,8 +8,9 @@ for (var idx in fields) {
 _%>
 import { Component, OnInit, OnDestroy } from '@angular/core';
 import { ActivatedRoute } from '@angular/router';
+import { Subscription } from 'rxjs/Rx';
 <%_ if (enableTranslation) { _%>
-import { JhiLanguageService<% if (fieldsContainBlob) { %>, DataUtils<% } %> } from 'ng-jhipster';
+import { EventManager, JhiLanguageService<% if (fieldsContainBlob) { %>, DataUtils<% } %> } from 'ng-jhipster';
```

Comment:
No need to add the `<% if (enableTranslation) { %>` as it will be stripped by `copyTemplate`